Sources/SwiftDocC/Model/DocumentationNode.swift

```diff
@@ -447,10 +447,14 @@ public struct DocumentationNode {
                     for: SymbolGraph.Symbol.Location.self
                 )?.url()
 
-                for comment in docCommentDirectives {
-                    let range = docCommentMarkup.child(at: comment.indexInParent)?.range
+                for directive in docCommentDirectives {
+                    let range = docCommentMarkup.child(at: directive.indexInParent)?.range
                     
-                    guard BlockDirective.allKnownDirectiveNames.contains(comment.name) else {
+                    guard BlockDirective.allKnownDirectiveNames.contains(directive.name) else {
+                        continue
+                    }
+
+                    guard DirectiveIndex.shared.renderableDirectives[directive.name] == nil else {
```

Comment:
Nit: Could we add a comment here explaining why we're allowing renderable directives?